python/test/joint_test.sdf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+    <world name="world_test">
+        <model name="model_test">
+
+            <link name="link_test_1">
+            </link>
+
+            <link name="link_test_2">
+            </link>
+
+            <joint name="joint_test" type="revolute">
+                <parent>link_test_1</parent>
+                <child>link_test_2</child>
+                <pose>0 0.5 0 0 0 0</pose>
+                <thread_pitch>2</thread_pitch>
+                <axis>
+                    <xyz>0 0 1</xyz>
+                </axis>
+                <sensor type="camera" name="sensor_test">
```

Comment:
nit: I think the only sensor type for joints is `force_torque`
```suggestion
                <sensor type="force_torque" name="sensor_test">
```

Reply:
Done.